### ISSUE TITLE
Include “potential reviews” in graph

### ISSUE THREAD

**Issue by smearedink** [davidsansome/tsurukame#403]
I find the graph of upcoming reviews very helpful, but slightly lacking in one way. Say I have a stack of 50 reviews coming up at 10am. Maybe 40 of those are old items where the next review (if I get them right) is well in the future, and 10 of them are new items where they’ll show up in my review stack one or more times again just today. But currently, before I’ve tackled that stack of 50, there’s no indication of those extra reviews I’ll have in a few hours.

I understand that there is a difference between reviews that are definitely coming and predicted reviews that will come only if you do prior reviews at the earliest possible time and get them right, but I think showing the latter (in a different colour bar, I’d think something fainter to suggest the uncertainty) would be nice. Basically, “here are the reviews you’ll be doing if you do the definitely-scheduled ones at the earliest possible time and get them correct”. So if you wait and let reviews pile up, these prediction bars will also just shift back in time.

**Comment by UInt2048** [comment]
Good idea - would be a great part of the #112 overhaul. Should be a setting though!
